### PR TITLE
Update some of lisp ports

### PR DESCRIPTION
**Pull request by catap**
#### Description

Here the first part of update lisp ports.

###### Type(s)

- [x] enhancement

###### Tested on

macOS 12.7.1 21G920 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: lisp/cl-local-time/Portfile [hunk 1]
```diff
@@ -35,4 +35,6 @@ subport cl-postgres-plus-local-time {
     common_lisp.threads yes
 
     common_lisp.systems {cl-postgres+local-time.asd}
+
+	livecheck.type      none
```
Comment (l2dy) [inline lisp/cl-local-time/Portfile:39]: Please convert tab to spaces.

File: lisp/cl-postmodern/Portfile [hunk 2]
```diff
@@ -33,8 +33,7 @@ subport cl-postgres {
     common_lisp.systems \
                     cl-postgres.asd
 
-    patchfiles-append \
-                    ecl-usocket.diff
+	livecheck.type  none
```
Comment (l2dy) [inline lisp/cl-postmodern/Portfile:36]: Same here.

**Review by l2dy — APPROVED** [review]
LGTM

**Comment by catap** [comment]
@l2dy May I ask you to merge it? I’m maintener of all lisp ports 🫣

**Comment by l2dy** [comment]
Sure. I was waiting for the CI to pass.

**Comment by catap** [comment]
Thanks!

**Pull request merged**